src/Samples/AuthoringDemo/AuthoringDemo/AuthoringDemo.csproj

```diff
@@ -3,15 +3,13 @@
     <PropertyGroup>
         <TargetFramework>net5.0-windows10.0.19041.0</TargetFramework>
         <Platforms>x64</Platforms>
-        <AssemblyVersion>1.0.0.0</AssemblyVersion>
     </PropertyGroup>
 
     <!-- CsWinRT properties -->
     <PropertyGroup>
         <CsWinRTComponent>true</CsWinRTComponent>
         <CsWinRTWindowsMetadata>10.0.19041.0</CsWinRTWindowsMetadata>
         <CsWinRTEnableLogging>true</CsWinRTEnableLogging>
```

Comment:
Probably comment out enable logging as folks should only need to enable that when filing bugs.